index.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable no-undef */
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] Let's remove the disable rules from every file, especially because you did a great job and there shouldn't be any linter errors. If there are, please make sure to fix them instead of disabling the rules. Remember that these are here to help us keep the code clean and standardized across teammates or the organization. Disabling them defeats their purpose. You can read more about [JavaScript best practices here.](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md)

Reply:
Thank you so much Daniel for this insight, we forgot to remove that